tmt/steps/execute/__init__.py

```diff
@@ -709,6 +715,7 @@ def go(
         logger: tmt.log.Logger,
     ) -> None:
         self.go_prolog(logger)
+        logger.verbose('ignore-duration', self.data.exit_first, 'green', level=2)
```

Comment:
```suggestion
        logger.verbose('ignore-duration', self.data.ignore_duration, 'green', level=2)
```
copy-paste leftover?

Reply:
:facepalm:

Reply:
thanks for spotting this, fixed in 2051375e